UndertaleModTool/Editors/UndertaleRoomEditor.xaml.cs

```diff
@@ -44,6 +45,10 @@ public partial class UndertaleRoomEditor : DataUserControl
 
         public static readonly PropertyInfo visualOffProp = typeof(Canvas).GetProperty("VisualOffset", BindingFlags.NonPublic | BindingFlags.Instance);
         private static readonly MainWindow mainWindow = Application.Current.MainWindow as MainWindow;
+        private static readonly Regex endNumberEx = new(@"\d+$", RegexOptions.Compiled);
+
+        // used for the flashing animation
+        public static Dictionary<UndertaleObject, FrameworkElement> ObjElemDict { get; } = new();
```

Comment:
Bit of a nitpick and out of scope for this PR, but this along with the other dicts here should get renamed to have clearer names.

Reply:
I'll consider it,